Assets/HoloToolkit-Examples/SpatialUnderstanding/SpatialUnderstanding-FeatureOverview/Scenes/SpatialUnderstandingExample.unity

```diff
@@ -1748,7 +1748,7 @@ Prefab:
         type: 2}
       propertyPath: RoomModel
       value: 
-      objectReference: {fileID: 100006, guid: c35ec8112ca03e04e904b72932bc97c5, type: 3}
+      objectReference: {fileID: 100008, guid: 68a7a806bf726614aa309bb190ab7cee, type: 3}
```

Comment:
Looks like "HoloToolkit\SpatialMapping\Tests\Scenes\SpatialProcessing.unity" also uses it.  I'm fine leaving it in so as not to disturb that scene (or other folks who may have added it to their own private test scenes), but I'd also be fine removing it and changing SpatialProcessing.unity to also use the new fbx I added.

Thoughts?